### PR TITLE
add port mapping and more specific docker examples

### PR DESCRIPTION
**Pull request by ldemailly**
```release-note
NONE
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,10 +11,12 @@ The name fortio comes from greek φορτίο which is load/burden.
...
-
-`docker run istio/fortio arguments...`
+Or use docker, for instance:
 
+```shell
+docker run -p 8080:8080 -p 8079:8079 istio/fortio server &
+docker run istio/fortio load http://$HOSTNAME:8080/
+```
 
 ## Command line arguments
```
Comment (andraxylia) [inline README.md:22]: Typo below: Fortio can be *and* http or gr - replace and with an

Comment (andraxylia) [inline README.md:22]: I would split command line arguments into 3 bullets, for clarity.

**Review by andraxylia — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (ldemailly) [reply README.md:22]: thx ! fixed!

Comment (ldemailly) [reply README.md:22]: not sure I understand this point, but I added bullets for the 3 types of example if that's what you meant - lmk

**Pull request merged**